osu.Game/Skinning/LegacySkin.cs

```diff
@@ -81,15 +89,31 @@ public override Drawable GetDrawableComponent(string componentName)
                         };
             }
 
-            // temporary allowance is given for skins the fact that stable handles non-animatable items such as hitcircles (incorrectly)
-            // by (incorrectly) displaying the first frame of animation rather than the non-animated version.
-            // users have used this to "hide" certain elements like hit300.
-            var texture = GetTexture($"{componentName}-0") ?? GetTexture(componentName);
+            var texture = GetTexture($"{componentName}-0");
 
-            if (texture == null)
-                return null;
+            if (texture != null && animatable)
+            {
+                var animation = new TextureAnimation { DefaultFrameLength = animationData.frametime };
+
+                for (int i = 1; texture != null; i++)
```

Comment:
What about the 0-th frame? Shouldn't it be included?

Reply:
``texture`` already contains the 0-th frame from the first animatable check. If you look at the loop body you can see that loading and adding is reversed exactly because I still have it in that variable.